doc/nrf/doc_build_process.rst

```diff
@@ -0,0 +1,78 @@
+.. _doc_build_process:
+
+Documentation build process
+###########################
+Whenever the main branch of the `sdk-nrf`_ repository is updated or a pull request is made, a new version of the documentation is built and published. The following section describes the steps involved with building and publishing the documentation.
+
+Building with Sphinx
+********************
+The |NCS| documentation uses the Sphinx framework to build the docsets for HTML.
+Sphinx primarily uses three files to work:
+
+* A makefile that runs the :code:`sphinx-build` command in the correct folder. This folder should contain the next two files.
+* A file :file:`index.rst` that serves as the top of the ToC tree and the root of the documentation.
```

Comment:
`At least one rst file that....` (index.rst name can be configured, or could even be a markdown file in MyST)

Reply:
Rewritten to only require an entry point